packages/compass-aggregations/src/components/pipeline-toolbar/index.tsx

```diff
@@ -127,8 +129,9 @@ export const PipelineToolbar: React.FunctionComponent<PipelineToolbarProps> = ({
 };
 
 const mapState = (state: RootState) => ({
-  isBuilderView: state.workspace === 'builder',
   isAIInputVisible: state.pipelineBuilder.aiPipeline.isInputVisible,
+  isBuilderView: state.workspace === 'builder',
+  isResultsMode: state.workspace === 'results',
```

Comment:
When you add `isResultsMode` as a separate prop, you basically allow to do `<PipelineToolbar isResultMode isBuilderView />` when rendering unconnected component which is not an expected behavior, this is a pretty confusing interface without relying on the knowledge of the store. The ticket is also about rendering this value only when in builder mode, the prop that is already being passed here. Let's clarify this interface and remove unnecessary ambiguity

Reply:
I don't have a preference, one is `workspace` state, another is `pipelineBuilder.pipelineMode` state, that seems to be a good distinguisher from my perspective, but feel free to change it if you want to